plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/nessie/TestIcebergNessieCatalogConnectorSmokeTest.java

```diff
@@ -72,6 +88,8 @@ protected QueryRunner createQueryRunner()
                                 "iceberg.catalog.type", "nessie",
                                 "iceberg.nessie-catalog.uri", nessieContainer.getRestApiUri(),
                                 "iceberg.nessie-catalog.default-warehouse-dir", tempDir.toString(),
+                                "iceberg.nessie-catalog.authentication.type", "BEARER",
+                                "iceberg.nessie-catalog.authentication.token", keycloakContainer.getAccessToken(),
```

Comment:
WDYT about adding a test with an invalid token just to make sure auth checks do work?

Reply:
I did try initially, `createQueryRunner` is common to all the tests. So, we are not initializing the catalog for each test. So, passing the wrong token for a single test or testing wrong token for whole suite needs lot of refactoring. 


I have manually verified it with the same framework by passing wrong/expired token. 

I will check if there is any way to add a new test class with the wrong token without duplicating much of the code.

Reply:
I have added a testcase now. `TestIcebergNessieCatalogWithAuth` checks with both valid and invalid token. 

I have also removed keycloak from `TestIcebergNessieCatalogConnectorSmokeTest` as it is an overkill and the new class tests the same.